### ISSUE TITLE
Review accessibility of the Telefónica brand palette

### ISSUE THREAD

**Issue by AnaMontes11** [Telefonica/mistica-design#2075]
## Description
It has been detected that the textSecondary color (grey5 #6E7894) in the Telefónica brand does not meet accessibility standards regarding contrast.

We should review the entire brand palette to:

- Ensure all colors comply with WCAG 2.2 guidelines for contrast (4.5:1 minimum for regular text and 3:1 for large text).
- Identify and document any colors that fail to meet accessibility standards.
- Propose adjustments or accessible alternatives where needed.

<img width="745" alt="Image" src="https://github.com/user-attachments/assets/d05d6638-9030-46be-95e4-d75579f88a35" />

## Tasks

- [ ] Android ticket
- [ ] iOS ticket
- [ ] Web ticket
